database/mongodb/transformData.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+const csv = require('csv-parser');
+const path = require('path');
+const client = require('./index.js');
+const { Product, Style } = require('./models.js');
+
+const db = mongoose.connection;
+
+
+async function transformData() {
+  try {
+    db.on('error', console.error.bind(console, 'MongoDB Connection error:'));
+    db.once('open', async () => {
+      const csvFiles = ['product.csv', 'features.csv', 'related.csv'];
```

Comment:
Processing product.csv here and also in index-mongodb. Probably only need to import it once.